### ISSUE TITLE
Clicking query input field when scrolled down jumps screen upwards

### ISSUE THREAD

**Issue by roganartu** [grafana/grafana#11275]
### Steps
- Create a panel with multiple input fields (enough to scroll the screen downwards).
- Expand query input fields
- Click between them multiple times

It appears inconsistent. I have some dashboards where I can almost never click without it jumping, and others that it only jumps occasionally.

Screencap: https://media.giphy.com/media/8PvzTW38DuPsui57Vd/source.mp4

### Versions
Tested on
v5.0.1-pre1 (commit: b1494637d)
v5.1.0-pre1 (commit: 1014f7c40)

Both Chrome (62.0.3202.62) and Firefox (58.0.2)

**Comment by pl-kabaradjian** [comment]
Hello @roganartu and @bergquist,
I would like to add some content to this issue as it has a huge impact on user experience for me, especially if you have long queries. I even ended up not using the query editor (I copy paste from another editor even for small changes).

If it can helps, I noticed that when it happens it scrolls up in such way that the bottom of my screen is the first line of the query I clicked.

**Version**
v5.2.1 (2040f61)

On Firefox 62.0b18
